server/odc-core/src/main/resources/i18n/BusinessMessages_zh_TW.properties

```diff
@@ -900,3 +900,6 @@ com.oceanbase.odc.PartitionPlanVariableKey.INTERVAL=間隔
 com.oceanbase.odc.PartitionPlanVariableKey.LAST_PARTITION_VALUE=最後一個分區規則對應位置表達式的值
 com.oceanbase.odc.partitionplan.TimeDataType=時間類型
 com.oceanbase.odc.partitionplan.NumberDataType=數字類型
+
+# MockData Task Message
+com.oceanbase.odc.MockData.null.message=未能獲取到{0}，任務還未開始執行或者已完成
```

Comment:
this is not a specific problem for mock data, we need to use a common error code to process this situation~